Added Appsmith integration doc

danciaclara opened the pull request:
## What kind of change does this PR introduce? 

This is a documentation update for connecting Supabase to Appsmith.

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

This is the first draft. Please feel free to add comments and we will update the doc accordingly.

danciaclara committed:
Added Appsmith integration doc

Isaiah-Hamilton reviewed (commented):
Make sure to add a link to the appsmith docs in the sidebar

Isaiah-Hamilton commented on web/docs/guides/integrations/appsmith.mdx:
```diff
@@ -0,0 +1,144 @@
+
+# Step 1: Set up your Backend on Supabase
+
+- On the Supabase dashboard, click `New project` and set the name to **Support Dashboard**
```
Link to the supabase dashboard for ease of use.

```suggestion
- On the [Supabase dashboard](https://app.supabase.io), click `New project` and set the name to **Support Dashboard**
```

Isaiah-Hamilton commented on web/docs/guides/integrations/appsmith.mdx:
```diff
@@ -0,0 +1,144 @@
+
+![project-settings-supabase-03](/img/guides/integrations/appsmith/project-settings-supabase-03.png)
+
+- Go to **Datasources** > **Create New** > **PostgreSQL**
```
Let people know that this is on appsmith's dashboard. This can be reworded if you want, but make sure that people know that this is in appsmith.
```suggestion
- Go to appsmith **Datasources** > **Create New** > **PostgreSQL**
```

Isaiah-Hamilton commented on web/docs/guides/integrations/appsmith.mdx:
```diff
@@ -0,0 +1,144 @@
+
+- Add widgets to the **Home** tab to create the dashboard as shown in the screenshot below. For eg: **Critical Open Issues** is a **Text** widget and below it is an **Input** widget which we will bind later to display the number of open tickets.
+
+[https://www.notion.so](https://www.notion.so)
```
Any reason for this here?

danciaclara committed:
Added Appsmith on sidebar, Added Supabase link, few other edits

danciaclara commented:
@Isaiah-Hamilton Thank you for reviewing the PR. I have updated with the requested changes. I have added Appsmith under the `sidebar.js` file. Do let me know if it needs to be added anywhere else.

Isaiah-Hamilton reviewed (approved):
lgtm
